module/behaviour/skills/Getup/src/Getup.cpp

```diff
@@ -65,19 +65,34 @@ namespace module::behaviour::skills {
             EXECUTION_PRIORITY = file["EXECUTION_PRIORITY"].as<float>();
         });
 
-        fallenCheck = on<Trigger<Sensors>, Single>().then("Getup Fallen Check", [this](const Sensors& sensors) {
-            // check if the orientation is smaller than the cosine of our fallen angle
-            if (!gettingUp && fabs(sensors.Htw(2, 2)) < FALLEN_ANGLE) {
-                updatePriority(GETUP_PRIORITY);
-                fallenCheck.disable();
-            }
-        });
+        fallenCheck = on<Last<20, Trigger<RawSensors>>, Single>().then(
+            "Getup Fallen Check",
+            [this](const std::list<std::shared_ptr<const RawSensors>>& sensors) {
+                Eigen::Vector3d acc_reading = Eigen::Vector3d::Zero();
+
+                for (const auto& s : sensors) {
+                    acc_reading += Eigen::Vector3d(s->accelerometer.x, s->accelerometer.y, s->accelerometer.z);
+                }
+                acc_reading.normalize();
```

Comment:
The more I think about it, the more I feel like we actually need to do this
```suggestion
                acc_reading = (acc_reading / double(sensors.size())).normalized();
```